pymc3/stats.py

```diff
@@ -509,6 +509,9 @@ def df_summary(trace, varnames=None, stat_funcs=None, extend=False, include_tran
         else:
             varnames = [name for name in trace.varnames if not name.endswith('_')]
 
+    if batches is None:
+        batches = min([100, len(trace)])
```

Comment:
ah, sorry, I had a bad memory  -- I knew `trace` was a dictionary-like object, so I thought `len` would be the number of keys.  Turns out it is overridden, so you're getting what is intended.